### DISCUSSION TITLE
Einstellungen DPL

### DISCUSSION THREAD

**Discussion by Narcodoc** [hoylabs/OpenDTU-OnBattery#699] [Q&A]
Wie muss ich die Einstellungen machen für einen Dynamic Power limiter. Mein hms 1600 regelt immer komplett runter und schreibt durch Fernsteuerung ausgeschaltet?
2x 425 Watt am HMS.
Ich habe zwar auch einen Speicher B2500 aber der lässt sich nicht mit dem Opendtu regeln.
Vielen Dank

**Comment by spcqike** [comment]
hast du beides angeschlossen? also batterie und Solarmodule?

aktiviere mal die ausführliche Protokollierung im DPL und guck in doe Konsole, wenn der Wechselrichter gestoppt wird. davon bräuchten wir einen Log zum Auswerten.

**Reply by Narcodoc** [reply]
Ich kenne mich noch nicht so gut aus. Ich schaue nach demLog

Ich habe 2 Paneele am Speicher mit eigenem BMS und von dort gehen2 Ausgänge zum Hms1600 4t.
Man kann ein Panel durchschleifen und wenn der Speicher voll ist schleift er komplett durch..
